### DISCUSSION TITLE
Parse ISO 8601 Duration to duration object

### DISCUSSION THREAD

**Discussion by Hazmi35** [js-fns/date-fns#2064] [Q&A] [answered]
I'm trying to parse [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) from YouTube Data API V3, but I don't seem to find any function that resolves ISO 8601 Durations into a duration object, is there any function that can do that?

**Comment by Hazmi35** [comment, marked as answer]
Okay, this feature is being worked on. #1947, I'm looking forward to it! I'll mark this as resolved

**Reply by bbigras** [reply]
continued in #2302

**Reply by davetapley** [reply]
and now in #3150
